### ISSUE TITLE
The Badges documentation needs to include visible textual equivalents of the meaning of the various badge background colors

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4562]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/badges

[WCAG 2.1 Guideline]
Success Criterion 1.4.1 Use of Color
(Level A)
https://www.w3.org/TR/WCAG21/#use-of-color

[OS]
any
[User Agents/Assistive Technologies]
NA

[Component]
Badges 

[Description]
The Badges documentation needs to include visible textual equivalents of the meaning of the various badge background colors

For example under "1. Badges (Color Options)" there are several colored badges but there no is indication of their meaning. If the badge color is purely decorative then we should mention that so readers understand why no textual equivalent for the badge styling is being presented. 

Under "2. Status Badges", the badge colors mean info, success, warning and danger according to the CSS classes badge-info, badge-success, badge-warning and badge-danger. But none of those terms are on the page next to the badges. 

User Impact]
Color blind users will not be able to perceive the meaning meant to be shared by the use of the the given color e.g. "danger". 

[Steps to Reproduce]
- Go to the Test URL 
- Observe that several badges lack visible textual equivalents
- Imagine how these badges would look without color, what would they mean?

[Recommendation]
- Provide visible text as an equivalent to the color
- Use cross-hatching or other non color based techniques along with documentation of what these techniques mean, similar to a key for a map.
